common/script_values/00_men_at_arms_values.txt

```diff
@@ -67,6 +67,12 @@ heavy_cavalry_recruitment_cost = @[heavy_cavalry_recruitment_cost]
 heavy_cavalry_low_maint_cost = @[heavy_cavalry_low_maint_cost]
 heavy_cavalry_high_maint_cost = @[heavy_cavalry_high_maint_cost]
 
+#Warcraft
+
+riflemen_recruitment_cost = @[archers_recruitment_cost * 2]
```

Comment:
How you calculated it. 55 * 2 = 110. On your balance spreadsheet, its cost is 120.